extmod/network_wiznet5k.c

```diff
@@ -63,6 +63,10 @@
 #include "lwip/dhcp.h"
 #include "netif/etharp.h"
 
+#if LWIP_IPV6
+#include "lwip/ethip6.h"
+#endif
```

Comment:
I suggest to just include this header unconditionally.  That header has its own `#if LWIP_IPV6` guard.  That keeps the code here a tiny bit simpler / easier to read.